backend/apps/skills_matching/views.py

```
@@ -0,0 +1,35 @@
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+from .models import ContributorProfile, SkillTag, IssueSkillTag, NewcomerFriendlinessScore, Recommendation, SkillGapAnalysis
+from .serializers import ContributorProfileSerializer, SkillTagSerializer, IssueSkillTagSerializer, NewcomerFriendlinessScoreSerializer, RecommendationSerializer, SkillGapAnalysisSerializer
+
+class ContributorProfileViewSet(viewsets.ModelViewSet):
+    queryset = ContributorProfile.objects.all()
+    serializer_class = ContributorProfileSerializer
+    permission_classes = [IsAuthenticated]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)